modular_skyrat/master_files/code/modules/antagonists/traitor/balancing.dm

```diff
@@ -0,0 +1,31 @@
+// Uplinks
+/obj/item/multitool/uplink/Initialize(mapload, owner, tc_amount)
+	. = ..()
+	tc_amount = 10
```

Comment:
for all of them, this
```suggestion
	tc_amount = 10
	var/datum/component/uplink/uplink = GetComponent(/datum/component/uplink)
	uplink.set_telecrystals(10)
```

Reply:
Strangely this didn't work for the uplinks besides the implant, but I found an alternative that looks fine, too.